content/efcore/getting-started/tutorial-database-reference.md

```diff
@@ -1,4 +1,5 @@
-# Database Tables 
+# Database Reference
+Below are the tables and data that these tutorials work with.
```

Comment:
Let's not say "these tutorials". Technically, all of the EF Core content is a single tutorial. Let's word it something like, "Here is the data contained within the example database used throughout this tutorial".